pylinac/winston_lutz.py

```diff
@@ -115,6 +115,23 @@ def __init__(self, directory: str, use_filenames: bool = False):
         images : :class:`~pylinac.winston_lutz.ImageManager` instance
         """
         self.images = ImageManager(directory, use_filenames)
+        self._json = {}
+    
+    @classmethod
+    def rebuild_json(self):
```

Comment:
Not needed. Properties always compute on call by default.

Reply:
Agreed.